api/envoy/extensions/filters/http/jwt_authn/v3/config.proto

```diff
@@ -528,4 +529,25 @@ message JwtAuthentication {
   // <http://www.w3.org/TR/cors/#cross-origin-request-with-preflight>`_ regardless of JWT
   // requirements specified in the rules.
   bool bypass_cors_preflight = 4;
+
+  // A map of unique requirement_names to JwtRequirements.
+  // :ref:`requirement_name <envoy_v3_api_field_extensions.filters.http.jwt_authn.v3.PerRouteConfig.requirement_name>`
+  // in `PerRouteConfig` uses this map to specify a JwtRequirement.
+  map<string, JwtRequirement> requirement_map = 5;
+}
+
+// Specify per-route config.
+message PerRouteConfig {
+  oneof requirement_specifier {
```

Comment:
not need to be in this PR, but it makes sense to have JwtRequirement directly inlined here instead of referring to the map? Add a TODO?

Reply:
I considered that option to have all JwtRequirement inline in the Per-Route.  But it needs to include all referred "Providers" too in order to be self-contain.  I considered this as too much duplication, abandon this approach.